vllm/model_executor/openvino_model_loader.py

```diff
@@ -603,6 +603,7 @@ def get_model(model_config: ModelConfig,
             model_config.model,
             export=export,
             compile=False,
+            load_in_8bit=False,
```

Comment:
propose to have env var to enable it back.

If you are OK, let's also mention it here https://github.com/ilya-lavrenov/vllm/blob/openvino-model-executor/use_with_openvino.md